### PR TITLE
Hw oop 0

### PR DESCRIPTION
**Pull request by CodeWebWeaver**
Implemented requested methods make it possible to find user by email and check successful login or not.
 Code passed through mvn clean package

### CODE REVIEW

File: src/main/java/mate/academy/service/UserService.java [hunk 1]
```diff
@@ -14,7 +14,12 @@ public class UserService {
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
+    public static User findByEmail(String email) {
```
Comment (KseniiaMakarova) [inline src/main/java/mate/academy/service/UserService.java:17]:
```suggestion
    public User findByEmail(String email) {
```

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 2]
```diff
@@ -5,12 +5,20 @@ public class AuthenticationService {
...
-     * @param password - user's password
+     * @param emailEntered - user's email
+     * @param passwordEntered - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
-    public boolean login(String email, String password) {
-        return false;
+    public boolean login(String emailEntered, String passwordEntered) {
+        if (UserService.findByEmail(emailEntered) == null) {
```
Comment (KseniiaMakarova) [inline src/main/java/mate/academy/service/AuthenticationService.java:14]: create an object and call this method on an object

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 3]
```diff
@@ -5,12 +5,20 @@ public class AuthenticationService {
...
+            return false;
+        } else {
+            if (passwordEntered.equals(UserService.findByEmail(emailEntered).getPassword())) {
+                return true;
+            } else {
+                return false;
+            }
+        }
```
Comment (KseniiaMakarova) [inline src/main/java/mate/academy/service/AuthenticationService.java:22]:
```suggestion
            return false;
        } 
        return passwordEntered.equals(user.getPassword());
```

File: src/main/java/mate/academy/service/UserService.java [hunk 4]
```diff
@@ -14,7 +14,12 @@ public class UserService {
...
+        for (User oneInAllUsers : users) {
```
Comment (KseniiaMakarova) [inline src/main/java/mate/academy/service/UserService.java:18]:
```suggestion
        for (User user : users) {
```

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 5]
```diff
@@ -1,16 +1,24 @@
...
-    public boolean login(String email, String password) {
-        return false;
+    public boolean login(String emailEntered, String passwordEntered) {
+        UserService userService = new UserService();
+        User findedUser = userService.findByEmail(emailEntered);
+        if (findedUser == null) {
+            return false;
+        } else {
+            return passwordEntered.equals(findedUser.getPassword());
+        }
```
Comment (KseniiaMakarova) [inline src/main/java/mate/academy/service/AuthenticationService.java:22]:
```suggestion
        } 
        return passwordEntered.equals(findedUser.getPassword());
```
we don't need else after return

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/service/AuthenticationService.java (hunk 5 above)
Comment (Skarbey) [reply src/main/java/mate/academy/service/AuthenticationService.java]: Thanks for advice. I fixed it

**Pull request closed without merging**